lib/getPostcssResult.js

```diff
@@ -1,29 +1,27 @@
-/* @flow */
 'use strict';
 
 const dynamicRequire = require('./dynamicRequire');
 const fs = require('fs');
 const LazyResult = require('postcss/lib/lazy-result');
 const postcss = require('postcss');
+/** @type {import('postcss-syntax')} */
 let autoSyntax = null;
 
 const postcssProcessor = postcss();
 
-module.exports = function(stylelint /*: stylelint$internalApi*/) /*: Promise<?Object>*/ {
-	const options /*: {
-    code?: string,
-    codeFilename?: string,
-    filePath?: string,
-    codeProcessors?: Array<Function>,
-    syntax?: stylelint$syntaxes,
-    customSyntax?: string
-  }*/ =
-		arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : {};
+/**
+ * @param {import('stylelint').StylelintInternalApi} stylelint
+ * @returns {Promise<import('postcss').Result>}
+ */
+module.exports = function(stylelint) {
+	/** @type {import('stylelint').GetPostcssOptions} */
+	const options = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : {};
 
-	const cached /*: ?postcss$result*/ = stylelint._postcssResultCache.get(options.filePath);
+	const cached = options.filePath ? stylelint._postcssResultCache.get(options.filePath) : undefined;
```

Comment:
Why this ternary?

Reply:
You can’t pass undefined in Map get parameter